ambry-commons/src/main/java/com.github.ambry.commons/HelixNotifier.java

```diff
@@ -142,6 +144,32 @@ public void unsubscribe(String topic, TopicListener<String> topicListener) {
   }
 
   /**
+   * A helper method that gets a {@link HelixNotifier}. Each call of this method will generate a different
+   * {@link HelixNotifier} instance.
+   * @param storeConfig A {@link HelixPropertyStore} used to instantiate a {@link HelixNotifier}.
+   * @return A {@link HelixNotifier}.
+   */
+  public static HelixNotifier getHelixNotifier(HelixPropertyStoreConfig storeConfig) {
```

Comment:
who are the callers of this method? I feel that this should be one of the constructor.